fe/fe-core/src/main/java/org/apache/doris/statistics/HMSAnalysisTask.java

```diff
@@ -117,39 +116,14 @@ public void doExecute() throws Exception {
      * Get table row count and insert the result to __internal_schema.table_statistics
      */
     private void getTableStats() throws Exception {
-        // Get table level information. An example sql for table stats:
```

Comment:
Please keep this `if (isPartitionOnly) ` branch, this branch maybe useful for partition statistics.  I think just modify the else branch is ok for now.